### ISSUE TITLE
Keyboard Navigation fails on: Nodes that are "outlined" on two sides, when jumped into

### ISSUE THREAD

**Issue by sifran-github** [freeplane/freeplane#1064]
**Describe the bug**
(see title)

**To Reproduce**

1. open the specific map
2. select the mid node, and jump in
3. try to navigate from the mid node to both children of this mid node - you cant.
4. try to navigate from the mid node's children to its sibling - you cant

NOTE: IT WILL JUMP TO THE LAST SELECTED CHILD, AND YOU WILL NOT BE ABLE TO SELECT THE OTHER CHILD.

**Expected behavior**

- navigation to/between children should work, just like it is when this node is not jumped into.

**Mind map files**
https://github.com/freeplane/freeplane/files/10677064/test_new_version_v2.zip

**Screenshots**
![image](https://user-images.githubusercontent.com/5730355/228065765-3886d826-db34-4aea-93ef-70350c89cb1d.png)

**Version**
freeplane-1.11.1-pre24

**Comment by dpolivaev** [comment]
Preview 1.11.1_25
